src/components/EmptySection/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './styles.scss';
+
+import * as Icons from '@carbon/icons-react';
+
+interface EmptySectionProps {
+  icon: string;
+  title: string;
+  description: string;
+  padding?: number;
+}
+
+const EmptySection = ({
+  icon, title, description, padding
+}: EmptySectionProps) => {
+  const Icon = Icons[icon];
+  return (
+    <div className="empty-section-container" style={padding ? { padding: `${padding}px 1rem` } : { padding: '2.375rem 1rem' }}>
```

Comment:
I understand the needs of the padding, but using it directly on the style attribute is not a good practice, is there another way to implement this?